### PR TITLE
Add further seccomp groups

### PR DESCRIPTION
**Pull request by topimiettinen**
Get further seccomp group definitions from systemd.

### CODE REVIEW

File: src/fseccomp/syscall.c [hunk 1]
```diff
@@ -190,6 +283,382 @@ static const SyscallGroupList sysgroups[] = {
...
+#ifdef SYS_select
+	  "select"
+#endif
+	},
+	{ .name = "@ipc", .list =
+#ifdef SYS_ipc
+	  "ipc,"
+#endif
+#ifdef SYS_memfd_create
+	  "memfd_create,"
```
Comment (rusty-snake) [inline src/fseccomp/syscall.c]: also blocked from `memory-deny-write-execute`

**Comment by rusty-snake** [comment]
IDK if it has any effect, but the last syscall in some groups has a trailing comma and in some there is no.

File: src/fseccomp/syscall.c (hunk 1 above)
Comment (topimiettinen) [reply src/fseccomp/syscall.c]: Yes, but I don't think that's so much of a problem, especially with the exceptions in place.

**Comment by topimiettinen** [comment]
@rusty-snake Thanks for the review. I only found (and fixed) one instance of trailing comma, did you find more?

**Comment by rusty-snake** [comment]
I found this while generating new lists for syscalls.txt, `@default` has it also (because mincore is commented). I know that this is not from this PR.

**Comment by rusty-snake** [comment]
@topimiettinen Beside that I'm asking me for what `@system-service` is good and what it means. I'm wondering why it include `@chown` and `@resources` twices.
```
chown<-privileged
privileged,resources<-default
default<-system-services
```

**Comment by topimiettinen** [comment]
@rusty-snake `@system-service` can be used as a default whitelist for systemd services with typical system calls allowed. For firejail, which has strong focus for user apps instead of system services, it's probably not so interesting unless someone would like to use firejail instead of systemd for a service, or otherwise use `@system-service` as a starting point for constructing filters.

**Comment by rusty-snake** [comment]
From the point of view it makes sense, using for allow and not to deny. Thx.

But what is with the double includes?

**Comment by topimiettinen** [comment]
Including `@chown` in `@privileged` makes sense because these system calls are indeed privileged. But indeed `@default` and `@system-service` could be optimized by dropping the redundant groups. 

Systemd's `@default` is used differently to Firejail's `@default`, that's why they do not match and why there could be these accidental overlaps.

**Comment by rusty-snake** [comment]
If `@system-service` is primary for whitelisting, and `@default` primary for blacklisting, it's a bit problematic to include `@default` in `@system-service`, because `@default` includes also `@obsolete`.

**Pull request merged**